### PR TITLE
Bugfix/snapshot fix

### PR DESCRIPTION
**Pull request by alexkeating**
Add back snapshot boost

## Onboarding
Screen Shot 2021-12-07 at 10 31 09 AM
Screen Shot 2021-12-07 at 10 31 18 AM
Screen Shot 2021-12-07 at 10 34 15 AM

## Snapshots
Screen Shot 2021-12-13 at 9 50 55 AM

## Settings
Screen Shot 2021-12-13 at 9 51 29 AM

### CODE REVIEW

File: src/components/overviewCard.jsx [hunk 1]
```diff
@@ -103,12 +102,24 @@ const OverviewCard = ({ daoOverview, members, daoVaults }) => {
...
           </Button>
           <Button
+            mr={6}
             onClick={() => history.push(`/dao/${daochain}/${daoid}/proposals`)}
             value='proposals'
             title={getTitle(customTerms, 'Proposals')}
           >
             {`View ${getTerm(customTerms, 'proposals')}`}
           </Button>
+          {daoMetaData?.boosts?.SNAPSHOT?.active && (
```
Comment (skuhlmann) [inline src/components/overviewCard.jsx]: We now have a hook for this that check if a boost is active:

https://github.com/HausDAO/daohaus-app/blob/develop/src/hooks/useBoost.jsx

you can do something like:

```
const { isActive } = useBoost();

...

{isActive('SNAPSHOT') && (
```


example:
https://github.com/HausDAO/daohaus-app/blob/develop/src/components/proposalList.jsx

File: src/data/boosts.js [hunk 2]
```diff
@@ -238,17 +238,17 @@ export const CONTENT = {
...
-  //     { href: 'https://discord.gg/daohaus', label: 'Boost Support' },
-  //   ],
-  // },
+  SNAPSHOT: {
+    title: 'Snapshot Proposals',
+    description:
+      'Gasless voting for quicker, smaller decisions or just collecting signal.',
+    publisher: PUBLISHERS.DAOHAUS,
+    version: '0.5',
+    pars: ['AWAITING CONTENT'],
```
Comment (skuhlmann) [inline src/data/boosts.js]: We have content for this now.

checkout this doc:
https://hackmd.io/o7X2OFPASpGGBWYpLEfOYw?view
Screenshot from 2021-12-14 09-25-30

File: src/data/forms.js [hunk 3]
```diff
@@ -699,6 +701,41 @@ export const BOOST_FORMS = {
...
+        explorerLink: true,
+      },
+      error: {
+        icon: BiErrorCircle,
+        title:
+          "Error: No space found! Please use the .eth name in your space's url",
+        errorMessage: true,
+      },
+    },
+    nextFormHook: async ({ values }) => {
```
Comment (skuhlmann) [inline src/data/forms.js]: I think we need to revisit this.

The legos are intended to be expressed as static content, not functions that need to execute. We should look at this together and think of a different spot to get this or way to express without having the function in the lego. Maybe if they are more like the customValidations - see rageQuit:

```
  RAGE_QUIT: {
    customValidations: ['canRagequit', 'rageQuitMinimum', 'rageQuitMax'],
    id: 'RAGE_QUIT',
    title: 'Rage Quit',
    required: [],
    tx: TX.RAGE_QUIT,
    fields: [
      [
        FIELD.RAGE_QUIT_INPUT,
        {
          ...FIELD.RAGE_QUIT_INPUT,
          htmlFor: 'loot',
          label: 'Loot to Rage',
          name: 'loot',
        },
      ],
    ],
  },
  
  and TxContext.jsx 
  handleCustomValidation
  ```
  
  
  
  then formbuilder knows to run those if they are there?

**Review by skuhlmann — CHANGES_REQUESTED** [review]

File: src/data/forms.js (hunk 3 above)
Comment (alexkeating) [reply src/data/forms.js]: Yes, let's take a look at this together. There are a couple things making this tricky

**Review by skuhlmann — APPROVED** [review]

**Pull request merged**